cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -352,7 +386,15 @@ object SparkSQLExecuteImpl {
           val sb = new StringBuilder()
           Utils.dataTypeStringBuilder(other, sb)(col)
           // write the full length as an integer
-          hdos.writeUTF(sb.toString(), true, false)
+          val utf8String = UTF8String.fromString(sb.toString())
```

Comment:
Why cant we write the string as simple string obtained from complex type string builder directly as VARCHAR and write it as writeUTF?